test/jdk/java/lang/ProcessBuilder/Basic.java

```diff
@@ -2865,15 +2865,6 @@ static void THROWS(Class<? extends Throwable> k, Fun... fs) {
     static boolean isLocked(BufferedInputStream bis) throws Exception {
         Field lockField = BufferedInputStream.class.getDeclaredField("lock");
         lockField.setAccessible(true);
-        var lock = (jdk.internal.misc.InternalLock) lockField.get(bis);
-        if (lock != null) {
-            if (lock.tryLock()) {
-                lock.unlock();
-                return false;
-            } else {
-                return true;
-            }
-        }
```

Comment:
Can you check if java.base/jdk.internal.misc can be dropped from the `@modules` in the test description?